src/ch/blinkenlights/android/vanilla/SeekBarPreference.java

```diff
@@ -103,7 +106,17 @@ protected View onCreateDialogView()
 
 		SeekBar seekBar = (SeekBar)view.findViewById(R.id.seek_bar);
 
-		int maxValue = (PrefKeys.SHAKE_THRESHOLD.equals(getKey()) ? 300 : 150);
+		int maxValue;
+		switch (getKey()) {
+			case PrefKeys.SHAKE_THRESHOLD:
```

Comment:
I'd like to keep compatibility with older java versions ;-)
Could you change this into if/else? (-source 1.5 only supports INT on switch statements)